tests/pages/firefox/base.py

```diff
@@ -12,8 +12,11 @@
 class ScrollElementIntoView(object):
 
     def scroll_element_into_view(self, strategy, locator):
+        # scroll elements so they are not beneath the header
+        page = self.page if isinstance(self, Region) else self
+        offset = {'x': 0, 'y': -page.family_navigation.root.size['height']}
```

Comment:
Hmm... one drawback now is scroll_element_into_view assumes the family navigation element is on the page. Could we maybe pass a locator to it instead?